README.md

```diff
@@ -3,90 +3,121 @@ GOV.UK Frontend ·
+
+To install all components, run:
+`npm install --save @govuk-frontend/all`
+
+
+To install individual components (for example, a button), run:
+`npm install --save @govuk-frontend/button`
+
+You can find a list of all components in the [packages directory](https://github.com/alphagov/govuk-frontend/tree/master/packages). Each component has a README with installation and usage instructions.
+
+After you have installed GOV.UK Frontend the @govuk-frontend package will appear in your node_modules folder.
+
+## Import assets
 
-1) [NodeJS](https://nodejs.org/en/) installed
-2) [package.json](https://docs.npmjs.com/files/package.json) file
+Import styles into your main Sass file.
 
-This file is needed to install a GOV.UK Frontend NPM package in your application. Most often it resides in the root of your application. You can create this file interactively using a `npm init` command.
+To import all components, add the line:
+`@import "@govuk-frontend/all/all";`
 
+To import an individual component (for example, a button), add the line:
+`@import "@govuk-frontend/button/button";`
 
-## Install all components
 
-```
-npm install --save @govuk-frontend/all
-```
+To resolve your @import declarations you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) or [load paths](url here) in Ruby.
 
-## Install individual components
+Below is a code sample you can add to your gulp configuration file using the gulp-sass package:
+`.pipe(sass({ includePaths: 'node_modules/' }))`
 
-Install a button component
+To import images, configure your application to reference or copy the icons assets.
 
-```
-npm install --save @govuk-frontend/button
-```
+Below is a code sample you can add to your [express.js](https://expressjs.com/) configuration:
+`app.use('/public', express.static(path.join(__dirname, '/node_modules/@govuk-frontend/icons')))`
 
-All components can be found in the packages directory.
-Each component has a README with installation and usage instructions.
+## Usage
+
+Copy and paste code from the examples in the GOV.UK [Design System](https://govuk-design-system-production.cloudapps.digital/)  to use GOV.UK Frontend in your service.
+
+<a id="option2"></a> 
+
+# Option 2: Add assets manually
 
-## Quick start (using all components)
+## Download assets
 
-:rotating_light: **This will not work until packages are published to npm.** :rotating_light:
+Download the latest versions of the following assets and include them in your project:
 
-Include the CSS and JavaScript
+- [govuk-frontend/tree/master/dist/css](https://github.com/alphagov/govuk-frontend/tree/master/dist/css)
+- [govuk-frontend/tree/master/dist/js](https://github.com/alphagov/govuk-frontend/tree/master/dist/js)
+- [icons](https://github.com/alphagov/govuk-frontend/tree/master/dist/icons)
 
-```
+## Include assets
+
+Add the CSS and JavaScript code to your HTML template:
+
+~~~
```

Comment:
Can this be marked up so that [syntax highlighting](https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting) works?

    ```html
    <!DOCTYPE html>
    [...]
    ```

Reply:
Done